go.work.sum

```diff
@@ -409,8 +409,8 @@ github.com/grafana/gomemcache v0.0.0-20231023152154-6947259a0586 h1:/of8Z8taCPft
 github.com/grafana/gomemcache v0.0.0-20231023152154-6947259a0586/go.mod h1:PGk3RjYHpxMM8HFPhKKo+vve3DdlPUELZLSDEFehPuU=
 github.com/grafana/grafana-plugin-sdk-go v0.212.0/go.mod h1:qsI4ktDf0lig74u8SLPJf9zRdVxWV/W4Wi+Ox6gifgs=
 github.com/grafana/grafana-plugin-sdk-go v0.215.0/go.mod h1:nBsh3jRItKQUXDF2BQkiQCPxqrsSQeb+7hiFyJTO1RE=
+github.com/grafana/grafana-plugin-sdk-go v0.216.0/go.mod h1:FdvSvOliqpVLnytM7e89zCFyYPDE6VOn9SIjVQRvVxM=
 github.com/grafana/grafana/pkg/promlib v0.0.3/go.mod h1:3El4NlsfALz8QQCbEGHGFvJUG+538QLMuALRhZ3pcoo=
-github.com/grafana/grafana/pkg/promlib v0.0.4/go.mod h1:621zjsCaFtP7mqqaiMJRN9h3SXrywAMaIPkeHvq3jsk=
```

Comment:
Oh no this should not be removed as the `main` is using promlib `v0.0.4` now. 
Could you please run `go mod tidy` and `go work sync` and commit the result?

Reply:
Thank you for your help with this! I think @toddtreece is working on a solution for keeping go.work.sum updated. In the meantime, I appreciate the help identifying what should be in this file!

Reply:
This is quite confusing for me too :) I put it back here https://github.com/grafana/grafana/pull/84397/commits/26e0830bc47d25138b4b9b50f55a74f527bb1277 and the rest seems fine. Thanks for your patience and sorry for this confusion. We will have a better state hopefully soon.